course/assets/css/videojs_player.scss

```diff
@@ -32,10 +32,6 @@ $slider-color: $pink;
     margin: 2.1em 0.45em;
   }
 
-  .vjs-control {
-    width: 3.5em;
```

Comment:
I certainly don't fully understand the purpose of all the CSS here, but I *suspect* that the `width` value was supposed to guarantee some sort of minimum width.

I tried `min-width: 3.5em` and that seemed to negate the need for the padding change below. (And avoiding the padding change might be good. It seemed to me that with the padding change dropped, the space around the "/" got bigger, and it's already weirdly big.)

Consider both of the points above minor suggestion... Priority is certainly making the whole time visible. 👍